code/modules/antagonists/clock_cult/items/clockwork_weapon.dm

```diff
@@ -0,0 +1,108 @@
+/obj/item/twohanded/clockwork
+	name = "Clockwork Weapon"
+	desc = "Something"
+	icon = 'icons/obj/clockwork_objects.dmi'
+	lefthand_file = 'icons/mob/inhands/antag/clockwork_lefthand.dmi';
+	righthand_file = 'icons/mob/inhands/antag/clockwork_righthand.dmi'
+	force_unwielded = 15
+	force_wielded = 5
+	block_power_wielded = 1
```

Comment:
this is power, not level. Power is percentage reduction of damage to stamina damage dealt. you want level

Reply:
block_power_wielded = 25
block_upgrade_walk = 1

Reply:
Not super sure how this works

Reply:
Iirc for wielded items it's block_level_wielded to make it block normally when wielded